content/knowledge_base/flare/consensus/avalanche/specification.md

```diff
@@ -0,0 +1,210 @@
+# Avalanche
+
+An internet-scale electronic payment system, evaluated in a large scale deployment, based on the Snow family of algorithms.
+The system's bottleneck is transaction verification.
```

Comment:
Added a longer and more friendly description here https://github.com/optakt/optakt.github.io/pull/13/commits/e0e27e6be8c0aef4153af5e2add9722380c23a59#diff-2d4f16e70781857aec6689a748ad155e45611b81f184aee076bb139890410b98R4-R5
Thanks! Please let me know if you think it is not clear enoough

Reply:
I left a comment on the commit, but since those sometimes get forgotten I'll add it here as well:

It's not forbidden, but [unusual to start a sentence with `But` in English](http://sites.utexas.edu/legalwriting/2020/05/01/beginning-with-but/#:~:text=There's%20no%20rule%20against%20beginning,But%20it%20rained%20that%20day.).

WDYT of writing something like this instead:

```diff
-But the validation of transaction scripts, UTXO references and related specifics is left to the application implementation.
+However, the validation of transaction scripts, UTXO references and related specifics is left to the application implementation.
```

Reply:
Yes, I remember that now, fixed here
https://github.com/optakt/optakt.github.io/pull/13/commits/dabd2c202a116853e67790cba4e525784f3d09d1
Thanks!